foundations/html_css/block-and-inline.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-The MDN box model article linked in the previous lesson mentions that different display types have subtly different box models. It also mentions that you can change how a box is calculated by changing the `display` property. We will explore the different display values you can use further in this lesson.
+CSS has two types of boxes - `block` boxes and `inline` boxes. The type of box refers to how the box behaves in terms of page flow and in relation to other boxes on the page. The display property is used to define how an HTML element should be displayed on the webpage. We will explore the different display values you can use further in this lesson.
```

Comment:
I think the first sentence in the previous introduction was a smoother intro to this topic. Could you incorporate this sentence into the new introduction?